src/frontend/Errors.hs

```diff
@@ -0,0 +1,13 @@
+module Errors where
+
+import Utils
+
+data Error = Error String Position
+    deriving Show
+
+data ErrorCategory = LexError | SemanticError
```

Comment:
Shouldn't we also have ParserErrors?

Reply:
Yes, but that can be a new issue. Currently the parser log parser and semantic errors as if they are the same thing.

Reply:
Will do!